### ISSUE TITLE
[C] Na liście zmienionych plików pokazać ilość komentarzy w danym pliku

### ISSUE THREAD

**Issue by fracz** [mobilecodereviewer/MobileCodeReviewer#36]
Jakby się dało, można by też dodać informację ile jest komentarzy w ogóle a ile jest nowych (w Webie się to robi ale nie wiem jak).

**Comment by d00d171** [comment]
Nowych komentarzy, czyli draftów?

**Comment by fracz** [comment]
Nie, komentarzy od innych reviewerów, których jeszcze nie widziałem. Gerrit w webie to jakoś rozpoznaje (podaje takie info na liście zmienionych plików). Jak nie da się tego prosto zrobić to wystarczy ilość komentarzy dodanych do pliku w ogóle.
